app/src/main/java/com/mobile/tandil/javabaseproject/mvp/model/ParkingModel.java

```diff
@@ -3,4 +3,16 @@
 import com.mobile.tandil.javabaseproject.mvp.contract.ParkingContract;
 
 public class ParkingModel implements ParkingContract.Model{
+
+    private int parkingAvailable = 0;
```

Comment:
Use an invalid value as default, for example -1

Reply:
Done